src/utils/webauthn.js

```diff
@@ -0,0 +1,159 @@
+import base64url from 'base64url'
+import {Serialize, Numeric} from 'eosjs'
+import { ec } from 'elliptic'
+
+// taken from the
+const formatWebauthnPubkey = (pubkey) => {
+  const clientDataStr = String.fromCharCode.apply(null, new Uint8Array(pubkey.clientDataJSON))
+  return {
+    attestationObject:  base64url.encode(pubkey.attestationObject),
+    clientData: JSON.parse(clientDataStr)
+  }
+}
+
+const decodeWebauthnSignature = (assertion, key) => {
```

Comment:
A lot of this black box "magic" code feels like it should be in its' own util repo, especially if people are going to be using this regularly in the future.